### ISSUE TITLE
PBFT batch should be defined as default

### ISSUE THREAD

**Issue by binhn** [hyperledger-archives/fabric#1338]
Currently batch is a configurable mode and the default is `batchsize: 2`, which is not reasonable default. We should set the default to be batch and size around 500; no magic about this number, but it is supposed to be batched for better performance :-)

**Comment by tuand27613** [comment]
A few of us are thinking about combining batch and classic into one codebase with classic becoming the special case where batchsize=1   

@corecode  @jyellick 

Also,  now that we've merged PR #1325,  trying to find some time to do performance measurements at different batch sizes

**Comment by corecode** [comment]
I don't know whether the number of transactions is a good metric to limit a batch.  If the reason is to limit the size of the resulting block, we should limit by size, not by count.

**Comment by corecode** [comment]
In any case, this is not a bug.

**Comment by binhn** [comment]
Is it better performance with batch = 2 or batch > 2 ?

**Comment by ibmmark** [comment]
I agree with @corecode this should be removed from the bug list and tagged as consensus.
